package/MDAnalysis/core/groups.py

```diff
@@ -631,6 +638,7 @@ def center(self, weights, pbc=None):
         If the :class:`MDAnalysis.core.flags` flag *use_pbc* is set to
         ``True`` then the `pbc` keyword is used by default.
 
+        .. versionchanged:: 0.19.0 Added `compound` parameter
         """
         atoms = self.atoms
         if pbc is None:
```

Comment:
PBC handling using `pack_into_box()` should go to the case where `compound.lower() == 'group'`.